src/app/feature/consultant/components/consultant-picture-edit/consultant-picture-edit.component.ts

```diff
@@ -48,9 +50,15 @@ export class ConsultantPictureEditComponent implements OnDestroy {
         urlProfileImage: this.croppedImage
       };
 
-      this.consultantStore.updateConsultant(updatedData)
-        .pipe(takeUntil(this.destroy$))
-        .subscribe(() => this.close());
+      if(this.croppedImage){
```

Comment:
I think this if can be removed because if `this.croppedImage.length > 0` on line 47 then `this.croppedImage` will return truthy.